starwars/src/App.css

```diff
@@ -4,5 +4,6 @@
 .Header {
```

Comment:
Instead of creating a Header class, you could use specificity to your advantage and do something like `.App h1 {...}` instead.